index.html

```diff
@@ -34,7 +34,7 @@
 </p>
 
 {% highlight bash %}
-zypper install yast2-devtools rubygem-yast-rake git-core
+zypper install yast2-devtools ruby2.1-rubygem-yast-rake git-core
```

Comment:
I was thinking more about rethinking the whole text to be more explicative than just a bash command. Something like
```
<p>
In a recent openSUSE, it's as simple as installing Git, the YaST development tools and the YaST-Rake gem.
The first two are easily installed running.
{% highlight bash %}
zypper install yast2-devtools git-core
{% endhighlight %}
<p>
In order to install Ruby gems for YaST development you should always install the package that corresponds
to the version of Ruby that is used by default in your system. For example, in systems based in Ruby 2.1,
the package name would be "ruby2.1-rubygem-yast-rake". The following set of commands will always install
the right package by first guessing your Ruby version and storing it in a variable (make sure there are not
spaces before and after "=").

{% highlight bash %}
ruby_version=`ruby -e "puts RbConfig::CONFIG['ruby_version']"`
zypper install -C "rubygem(ruby:$ruby_version:yast-rake)"
{% endhighlight %}
```

Something like that